package.json

```diff
@@ -1,5 +1,21 @@
 {
-  "name": "ux-onboarding-project",
+  "name": "nathan-phillips-ux-iop",
   "version": "1.0.0",
-
+  "main": "index.html",
+  "dependencies": {
+    "server": "^1.0.18"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "start": "http-server ./src/index.html -p 1820"
```

Comment:
This start script didn't work for me.
I ended up changing it to the following:
```json
    "start": "http-server ./src -p 1820"
```
Which serves the contents of the `src` folder, and the browser defaults to `index.html`

Reply:
did the file show up styled for you? It's actually not working for me now for some reason.